charts/pulsar/templates/zookeeper/zookeeper-configmap.yaml

```diff
@@ -45,6 +45,7 @@ data:
   {{- if .Values.zookeeper.reconfig.enabled }}
   PULSAR_PREFIX_reconfigEnabled: "true"
   PULSAR_PREFIX_quorumListenOnAllIPs: "true"
```

Comment:
I heard this one might not be needed and may cause problems, could you do some research about this config item?